components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -818,6 +818,42 @@ void update_host_dse_f(Int shcol, Int nlev, Real* thlm, Real* shoc_ql, Real* exn
   ekat::pack::device_to_host({host_dse}, {shcol}, {nlev}, inout_views, true);
 }
 
+void check_tke_f(Int shcol, Int nlev, Real* tke)
+{
+  using SHF = Functions<Real, DefaultDevice>;
+
+  using Scalar     = typename SHF::Scalar;
+  using Spack      = typename SHF::Spack;
+  using view_2d    = typename SHF::view_2d<Spack>;
+  using KT         = typename SHF::KT;
+  using ExeSpace   = typename KT::ExeSpace;
+  using MemberType = typename SHF::MemberType;
+
+  Kokkos::Array<view_2d, 1> temp_2d_d;
+
+  // Sync to device
+  ekat::pack::host_to_device({tke}, shcol, nlev, temp_2d_d, true);
+
+  view_2d
+    tke_d(temp_2d_d[0]);
+
+ const Int nk_pack = ekat::pack::npack<Spack>(nlev);
```

Comment:
You went from 2-space indent to 1.

Reply:
Thanks @jgfouca for noticing these indenting issues.